### DISCUSSION TITLE
Internal server error wehn deploying micro with deploy button

### DISCUSSION THREAD

**Discussion by Viiprogrammer** [deta/cloud-docs#399] [Help]
![IMAGE](https://user-images.githubusercontent.com/17622604/185803864-c92e25fb-6e44-4d76-8ffc-2ac588434428.png)

Repository https://github.com/Viiprogrammer/MDNBot

**Comment by lemonyte** [comment]
Hello Megumin, if you try to deploy the code to a Micro using the `deta deploy` command, are there any errors?

**Reply by Viiprogrammer** [reply]
Success deploy, no errors, work fine

```
righthaze@anime-desktop  ~/Documents/GitHub/MDNBot   main ±  deta new
Successfully created a new micro
{
	"name": "MDNBot",
	"id": "....",
	"project": "....",
	"runtime": "nodejs14.x",
	"endpoint": "https://.....deta.dev",
	"region": "eu-central-1",
	"visor": "disabled",
	"http_auth": "disabled"
}
Adding dependencies...

+ dotenv@16.0.1
+ node-fetch@2.6.7
+ express@4.18.1
+ cheerio@1.0.0-rc.12
+ telegraf@3.40.0
+ undici@5.9.1
+ telegraf-i18n@6.6.0
+ telegram-escape@1.1.1
added 93 packages from 68 contributors and audited 93 packages in 9.705s

18 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

righthaze@anime-desktop  ~/Documents/GitHub/MDNBot   main ±  deta visor enable
Successfully enabled visor mode
 righthaze@anime-desktop  ~/Documents/GitHub/MDNBot   main ±  deta deploy
Everything up to date
righthaze@anime-desktop  ~/Documents/GitHub/MDNBot   main ±  deta update -e .env
Updating environment variables...
Successfully updated micro's environment variables
```

**Reply by aavshr** [reply]
hi @Viiprogrammer sorry seems to be a bug with the UI deployment flow. 

Can you please try deploying once through the button without the `.detaignore` file and see if it works?

**Reply by Viiprogrammer** [reply]
> hi @Viiprogrammer sorry seems to be a bug with the UI deployment flow.
> 
> Can you please try deploying once through the button without the `.detaignore` file and see if it works?

I created a branch and delete `.detaignore`, it deploys fine

**Reply by aavshr** [reply]
@Viiprogrammer thank you. We will investigate and fix as soon as we can.
